src/current/_includes/v23.1/sidebar-data/reference.json

```diff
@@ -1340,14 +123,60 @@
             "urls": [
               "/cockroachcloud/pci-dss.html"
             ]
+          },
+          {
+            "title": "System Catalogs",
+            "items": [
+              {
+                "title": "Overview",
+                "urls": [
+                  "/${VERSION}/system-catalogs.html"
+                    ]
+              },
+              {
+                "title": "<code>crdb_internal</code>",
+                "urls": [
+                  "/${VERSION}/crdb-internal.html"
+                ]
+              },
+              {
+                "title": "<code>information_schema</code>",
+                "urls": [
+                  "/${VERSION}/information-schema.html"
+                ]
+              },
+              {
+                "title": "<code>pg_catalog</code>",
+                "urls": [
+                  "/${VERSION}/pg-catalog.html"
+                ]
+              },
+              {
+                "title": "<code>pg_extension</code>",
+                "urls": [
+                  "/${VERSION}/pg-extension.html"
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "title": "Spatial Features",
+        "items": [
+          {
+            "title": "Work with Spatial Data",
```

Comment:
This is intended to be combined with the Query Spatial Data page.

<img width="267" alt="image" src="https://github.com/cockroachdb/docs/assets/8740013/75b417c7-9154-40a6-a0f3-a40857fa9a57">

I assume this is planned for another phase of the IA rollout. Just wanted to call it out just in case. After that change is done, this section shouldn't exist under Reference.

Reply:
Yep, this will be done in the next phase.